security/index.adoc

```diff
@@ -63,16 +59,11 @@ application developer, the benefits of containers may sound slightly different:
 [[security-intro-further-reading-1]]
 ==== Further Reading
 
-- _{product-title} Architecture_:  xref:{sourcedir}architecture/core_concepts/containers_and_images.adoc#architecture-core-concepts-containers-and-images[Core Concepts -> Containers and Images]
+- _{product-title} Architecture_:  xref:../architecture/core_concepts/containers_and_images.adoc#architecture-core-concepts-containers-and-images[Core Concepts -> Containers and Images]
+-  link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/container_security_guide/[_Red Hat Enterprise Linux Atomic Host Container Security Guide_]
 
 [[security-intro-enterprise]]
-== Securing Containers for Enterprise Use
-
-Enterprise use of containers requires strong security. In many ways, securing
-containers is a lot like securing any running process. You need to think about
-security throughout the layers of the solution stack before you deploy and run
-your containers. You also need to think about security throughout the
-application and container life cycle.
+== Container Security in {product-title}
```

Comment:
I think if my suggestion above is taken in, then this might look weird. It's sort of a rehash of sorts of the intro para, so maybe the two can be merged?